src/datasources/network/network.module.spec.ts

```diff
@@ -1,10 +1,10 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { NetworkModule } from './network.module';
-import { RequestScopedLoggingModule } from '../../logging/logging.module';
+import { RequestScopedLoggingModule } from '@/logging/logging.module';
 import { ClsModule } from 'nestjs-cls';
-import { ConfigurationModule } from '../../config/configuration.module';
+import { ConfigurationModule } from '@/config/configuration.module';
 import configuration from '../../config/entities/__tests__/configuration';
```

Comment:
I've noticed a few places where the alias is not being used. Many of which are up and down a folder, but what do you think about beyond that?

Reply:
I assume there is a rule that allows relative imports up to a certain depth but I haven't found a way to auto replace those with the path alias. If I remove the import and import again it will be imported with a path alias though.

Reply:
I agree but we should align on this. @hectorgomezv, what are your thoughts?

Reply:
I've been researching this a bit because I've never used path aliases before. It seems nice, but I'd be in favor of going for an all-or-none strategy, mixing aliases with relative paths seems a bit ugly and error-prone to me. 

I'd wait for @fmrsabino opinion on this as well since is a wider change that would affect the whole team. I'm in fact trying to make my IDE understand it when dealing with auto-imports with no luck yet. As I said, I'm not against the change at all, but in my opinion, we could put this on hold and wait for the whole team to agree.

Reply:
Agree, we can split it further for each subdirectory. Regarding IDE support I had to configure both VSCode and Webstorm to work with this.

In Webstorm there is an option `Use paths relative to tsconfig.json` that I had to turn on.
In VSCode I had to change `typescript.preferences.importModuleSpecifier` to `non-relative` in the settings.json.

Reply:
You have some other cases like

```typescript
import { SafeAppsRepository } from '../../../../domain/safe-apps/safe-apps.repository';
```

https://github.com/safe-global/safe-client-gateway-nest/blob/e176080a51c2b8e17d6fa9ce48e283abf4c5d1f1/src/routes/transactions/mappers/common/safe-app-info.mapper.ts#L2-L4

For these cases, they'd become

```typescript
import { SafeAppsRepository } from `@domain/safe-apps/safe-apps.repository';
```

One disadvantage that I see with this approach though is that the modules are mapped via a regex in the testing configuration which looks like that it can break easily if we move somethings around 🤔

Reply:
Done: https://github.com/safe-global/safe-client-gateway-nest/pull/643 
If we agree to use this, we would need to modify that file 🙂

Reply:
> One disadvantage that I see with this approach though is that the modules are mapped via a regex in the testing configuration which looks like that it can break easily if we move somethings around 🤔

The mapping we do in the jest config is very similar to the mapping in the tsconfig. Ideally, it would be one source of truth for both mappings but I haven't found a way to do that.

Reply:
There is a way to disallow relative imports via eslint ("no-restricted-imports") but they would have to be fixed by hand afaik. We could still add an exception for imports from the same directory so if we decide to go with path aliases I can do that.

Reply:
I've added path aliases for domain, datasources and routes. As discussed with @fmrsabino, further restricting relative imports is something we can add later on.

Reply:
> I've added path aliases for domain, datasources and routes. As discussed with @fmrsabino, further restricting relative imports is something we can add later on.

Awesome! Thanks :) I will have a look into the new changes.